src/main/java/com/dropbox/core/android/AuthActivity.java

```diff
@@ -255,7 +292,21 @@ public static Intent makeIntent(Context context, String appKey, String webHost,
     public static Intent makeIntent(Context context, String appKey, String desiredUid, String[] alreadyAuthedUids,
                                     String sessionId, String webHost, String apiType) {
         if (appKey == null) throw new IllegalArgumentException("'appKey' can't be null");
-        setAuthParams(appKey, desiredUid, alreadyAuthedUids, sessionId, webHost, apiType);
+        setAuthParams(appKey, desiredUid, alreadyAuthedUids, sessionId, webHost, apiType, null,
+            null, null);
+        return new Intent(context, AuthActivity.class);
+    }
+
+    static Intent makeIntent(
+        Context context, String appKey, String desiredUid, String[] alreadyAuthedUids,
+        String sessionId, String webHost, String apiType, TokenAccessType tokenAccessType,
+        DbxRequestConfig requestConfig, DbxHost host
+    ) {
+        if (appKey == null) throw new IllegalArgumentException("'appKey' can't be null");
```

Comment:
I generally avoid one liner ifs without braces.  Better to be verbose and avoid bugs.  Not sure if that is the code style in here though.